### DISCUSSION TITLE
v2 OSS - Dev URLs?

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/coder#5972] [General] [answered]
Let's say I have a React app running on 3000 and an Express app on 3001. I want to talk to the Express app from the React app. If I do fetch `https://3001--dev--joe-coder--jsjoeio--apps.dev.coder.com/`, I get a CORS error because it's behind Coder's OAuth/Proxy.

What's the workaround? I know we had Dev URLs in v1. Is there something like that in v2? 

Otherwise the only workaround that comes to mind is something like `ngrok`.

**Comment by bpmct** [comment, marked as answer]
Ah, looks like this is a feature we need to add. #5706 

In the meantime, you can use VS Code Desktop and port-forwarding or `ngrok` I suppose.
